### PR TITLE
 To Segment the ISIC data set with the UNET, in accordance that the predicted segmentation response with the test data set should have a minimum dice co-efficient of 0.7

### PR DESCRIPTION
**Pull request by smitraDA**
Submitted By: Shobhan Mitra
Student ID:45522156

The objective of this project is to Segment the ISIC data set with the UNET,  in accordance that the predicted segmentation response with the test data set should have a minimum dice co-efficient of 0.7.

I have used ISIC dataset for training,validation and testing.I have trained model with UNET and best model has been saved on basis of lowest validation dice loss.

Finally a random ISIC test sample with lesion image, actual or ground truth segmentation image and predicted segmentation image has been plotted which was satisfactory.

### CODE REVIEW

**Comment by SiyuLiu0329** [comment]
- Sufficiently detailed commit messages
- Code valid for Easy Difficulty

Other Feedback
- Please group the `.png` files and put them in a sub-directory
- Please follow python file  and folder naming convention e.g. `lorem_ipsum_123_abc.py`

**Comment by rexsw** [comment]
Code looks good for easy difficulty

For next time;

Tensorflow has functions for image resizing and more so there’s no need to also use skimage and in your project. 

There’s no need to use a whole library for one function rather than using pandas you could have written that function yourself

**Comment by shakes76** [comment]
## TF Usage
OK
## Good Practice (Design/Commenting)
Good design and comments
## Algorithm 
Driver Script present
Shows Usage & Demo & Visualisation & Data usage, Readme OK
Module present
## Commit Log
Meaningful commit messages 
Progressive commits used 
## Documentation
Description and Comments good
Markdown used PDF submitted
## Pull Request
Successful Pull Request (Working Algorithm Delivered on Time in Correct Branch) 
**Feedback required. Move image files into sub directory.**
No Request Description. In the future, a more detailed pull request could've been done. When developer receiving the request is not known, the request could be rejected. Remember you could use/reuse the README info.
